$OpenBSD: patch-imap_sync_support_c,v 1.1 2017/05/29 11:56:33 ajacoutot Exp $

Unbreak if libical is found at configure time since we're not building http
support.

Index: imap/sync_support.c
--- imap/sync_support.c.orig
+++ imap/sync_support.c
@@ -60,7 +60,9 @@
 #include <limits.h>
 
 #include "assert.h"
+#ifdef USE_CALALARMD
 #include "caldav_alarm.h"
+#endif
 #include "global.h"
 #include "imap_proxy.h"
 #include "mboxlist.h"
